Makefile

```diff
@@ -221,6 +225,7 @@ run_tests.lighthouse_performance: ## Runs the lighthouse performance tests for t
 	$(MAKE) stop
 
 OS_NAME := $(shell uname)
+sharding_instances := 3
```

Comment:
What is this used for?

Reply:
this flags implements the shading instance flag. please refer -- https://github.com/oppia/oppia/blob/develop/scripts/run_e2e_tests.py#L59